### PR TITLE
Orthogonal Polynomials

### PR DESCRIPTION
**Pull request by 0x00b1**
```Python
chebyshev_polynomial_v(input, n, *, out=None) -> Tensor
```

Chebyshev polynomial of the third kind $V_{n}(\text{input})$.

```Python
chebyshev_polynomial_w(input, n, *, out=None) -> Tensor
```

Chebyshev polynomial of the fourth kind $W_{n}(\text{input})$.

```Python
legendre_polynomial_p(input, n, *, out=None) -> Tensor
```

Legendre polynomial $P_{n}(\text{input})$.

```Python
shifted_chebyshev_polynomial_t(input, n, *, out=None) -> Tensor
```

Shifted Chebyshev polynomial of the first kind $T_{n}^{\ast}(\text{input})$.

```Python
shifted_chebyshev_polynomial_u(input, n, *, out=None) -> Tensor
```

Shifted Chebyshev polynomial of the second kind $U_{n}^{\ast}(\text{input})$.

```Python
shifted_chebyshev_polynomial_v(input, n, *, out=None) -> Tensor
```

Shifted Chebyshev polynomial of the third kind $V_{n}^{\ast}(\text{input})$.

```Python
shifted_chebyshev_polynomial_w(input, n, *, out=None) -> Tensor
```

Shifted Chebyshev polynomial of the fourth kind $W_{n}^{\ast}(\text{input})$.

### CODE REVIEW

File: torch/special/__init__.py [hunk 1]
```diff
@@ -911,6 +918,34 @@
...
 
+chebyshev_polynomial_v = _add_docstr(_special.special_chebyshev_polynomial_v,
+                                     r"""
+chebyshev_polynomial_v(input, n, *, out=None) -> Tensor
+
+Chebyshev polynomial of the third kind :math:`V_{n}^{\ast}(\text{input})`.
+
+""" + r"""
+Args:
+    {input}
```
Comment (mruberry) [inline torch/special/__init__.py:929]: n is also an arg

File: torch/special/__init__.py [hunk 2]
```diff
@@ -911,6 +918,34 @@
...
+""".format(**common_args))
+
+chebyshev_polynomial_w = _add_docstr(_special.special_chebyshev_polynomial_w,
+                                     r"""
+chebyshev_polynomial_w(input, n, *, out=None) -> Tensor
+
+Chebyshev polynomial of the fourth kind :math:`W_{n}^{\ast}(\text{input})`.
+
+""" + r"""
+Args:
```
Comment (mruberry) [inline torch/special/__init__.py:942]: n is also an arg

**Review by mruberry — CHANGES_REQUESTED** [review]
Looks good overall but it's picked up a peachpy update I don't think we want?

There are also some small doc issues and this should update the .rst file to render the docs, too

**Comment by 0x00b1** [comment]
@mruberry Fixed the python-pypeach issue. :)

**Review by mruberry — APPROVED** [review]
Cool! Test failures are unrelated. Looking forward to the .rst update!

**Pull request closed without merging**